packages/mobile/src/escrow/saga.ts

```diff
@@ -333,36 +333,33 @@ function* withdrawFromEscrowWithoutCode(komenciActive: boolean = false) {
         value
       )
 
+      const withdrawAndTransferTx = mtwWrapper.executeTransactions([withdrawTx.txo, transferTx.txo])
+
       try {
         if (!komenciActive) {
-          const wrappedBatchTx = mtwWrapper.executeTransactions([withdrawTx.txo, transferTx.txo])
-          yield call(sendTransaction, wrappedBatchTx.txo, walletAddress, context)
+          yield call(sendTransaction, withdrawAndTransferTx.txo, walletAddress, context)
         } else {
           const komenciKit: KomenciKit = new KomenciKit(contractKit, walletAddress, {
             url: feelessVerificationState.komenci.callbackUrl || networkConfig.komenciUrl,
             token: feelessVerificationState.komenci.sessionToken,
           })
-          // TODO: When Komenci supports batched subsidized transactions, batch these two txs
-          // Currently not ideal that withdraw to MTW can succeed but transfer to EOA can fail but
-          // there will be a service in place to transfer funds from MTW to EOA for users
-          const withdrawTxResult: Result<CeloTxReceipt, FetchError | TxError> = yield call(
-            [komenciKit, komenciKit.submitMetaTransaction],
-            mtwAddress,
-            withdrawTx
-          )
 
-          if (!withdrawTxResult.ok) {
-            throw withdrawTxResult.error
-          }
+          const signedWithdrawAndTransferTx: CeloTransactionObject<string> = yield call(
+            [mtwWrapper, mtwWrapper.signAndExecuteMetaTransaction],
+            withdrawAndTransferTx.txo
+          )
 
-          const transferTxResult: Result<CeloTxReceipt, FetchError | TxError> = yield call(
+          const withdrawAndTransferTxResult: Result<
+            CeloTxReceipt,
+            FetchError | TxError
+          > = yield call(
             [komenciKit, komenciKit.submitMetaTransaction],
             mtwAddress,
-            transferTx
+            signedWithdrawAndTransferTx
           )
```

Comment:
I figured it out you don't need to use the sign and execute here.
You just need to pass in the `withdrawAndTransferTx` straight to `komenciKit.submitMetaTransaction` and that takes care of the signing and stuff. The way you did it now results in a double wrapped meta transaction that doesn't pass the filter.